pkg/operators/ebpf/ebpf.go

```diff
@@ -64,6 +65,7 @@ const (
 
 	// Keep in sync with `include/gadget/kernel_stack_map.h`
 	KernelStackMapName       = "ig_kstack"
+	UserStackMapName         = "ig_ustack"
 	KernelStackMapMaxEntries = 10000
```

Comment:
Name should be changed to something more general. This is not only the kernel stack map max entries now